torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -198,9 +198,18 @@ std::pair<IValue, c10::optional<IValue>> getFunctionTuple(
 
   // operators
   std::vector<IValue> operators;
+  auto op_to_specified_args = code.op_to_num_specified_args();
   operators.reserve(opnames.size());
   for (const auto& opname : opnames) {
-    operators.emplace_back(Tup({opname.name, opname.overload_name}));
+    auto unique_name = opname.overload_name != ""
+        ? opname.name + "." + opname.overload_name
+        : opname.name;
```

Comment:
Can this be refactored into a utility function so that it is shared by all places that need a unique name to prevent future divergence?